### PR TITLE
Added a 'help' pipeline that comments on a PR with available triggers

### PR DESCRIPTION
**Pull request by AverageMarcus**
Responds to PR comments with the `/run help` trigger and comments with all the available Pipelines in the cluster.

Blocked by:
* https://github.com/giantswarm/cicd-tools/pull/16
* https://github.com/giantswarm/cicd-tools/pull/17

### CODE REVIEW

**Review by ericgraf — APPROVED** [review]

File: tekton-resources/pipelines/help.yaml [hunk 1]
```diff
@@ -0,0 +1,138 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: help
```
Comment (ericgraf) [inline tekton-resources/pipelines/help.yaml:4]: missing namespace here.

File: tekton-resources/pipelines/help.yaml [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+  kind: ClusterRole
+  name: help-pipeline
+
+---
+
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: help
+  labels:
```
Comment (ericgraf) [inline tekton-resources/pipelines/help.yaml:48]: missing namespace? or do we leverage flux target namespace?

Comment (AverageMarcus) [reply tekton-resources/pipelines/help.yaml:48]: Exactly. It'll end up in tekton-pipelines. But as we have a CRB in this yaml we need a namespace anyway so I might as well be explicit. I'll update :)

Comment (AverageMarcus) [reply tekton-resources/pipelines/help.yaml:50]: Updated.

**Pull request merged**